components/osc-host-application/ApplicationForm.js

```diff
@@ -175,15 +171,11 @@ const messages = defineMessages({
   },
   linksToLicenses: {
     id: 'HostApplication.form.linksToLicenses',
-    defaultMessage: 'Link your license(s)',
-  },
-  linksToLicensesPlaceholder: {
-    id: 'HostApplication.form.linksToLicensesPlaceholder',
-    defaultMessage: 'You can add the links to your secondary or additional licenses here',
+    defaultMessage: 'Extra information about your license(s)',
```

Comment:
if we detect a good license is this field optional?

Reply:
For now, it's optional. Like anything in Code or Community.